source/extensions/filters/http/aws_request_signing/aws_request_signing_filter.h

```diff
@@ -67,28 +68,13 @@ class FilterConfigImpl : public FilterConfig {
 /**
  * HTTP AWS request signing auth filter.
  */
-class Filter : public Http::StreamFilter {
+class Filter : public Http::PassThroughFilter, Logger::Loggable<Logger::Id::filter> {
```

Comment:
Technically, this still doesn't need to be an encoder filter, so in a follow up if you feel like it I would split PassThroughFilter into PassThroughDecoderFilter, PassThroughEncoderFilter, and then make PassThroughFilter just combine both of those classes. But this is fine for now.

Reply:
Ah yes, will send a split fo that.

Reply:
Oops, I missed the fact that we already have a PassThroughDecoderFilter. Fix here:

https://github.com/envoyproxy/envoy/pull/9926